\section*{Methods & Results}

To demonstrate the merits of our recommendations, a number of assemblies were produced using a variety of methods. These assemblies were evaluated using methods descripbed elsewhere. 

All assembly datasets were produced by asembling a publically available 100bp paired-end Illumina dataset (Short Read Archive ID SRR797058, \citep{Macfarlan:2012js}). This dataset was subsetted randomly into 10, 20, 50, 75, and 100 million read pairs as described in \citep{MacManes:2014io}. Reads were quality trimmed using the software packare bless version 0.16 \citep{Heo:2014cb} and a kmer=19.  Adapters were removed from the reads, as were nucleotides with quality Phred $\leq$ 2, using the program Trimmmatic version 0.32 \citep{Bolger:2014ek}. Read error correction, when employed, was done using Seecer version 0.1.3 \citep{Le:2013dy}. The adapter and quality trimmed, error corrected reads were then assembled using Trinity release r0140717 or SOADdenovo-Trans version 1.03. Trinity was employed using default settings, while SOAPdenovo-Trans was employed after optimizing kmer size, [and those other flags i forget right now]. \\

All assemblies were characterized using Transrate version 0.31. 