\section*{Introduction}


For biologists interested in understanding the relationship between fitness, genotype, and phenotype, modern sequencing technologies provide for an unprecedented opportunity to gain a deep understanding of genome level processes that together, underlie adaptation. Transcriptome sequencing has been particularly influential, and as a direct result, a diverse toolset for the assembly and analysis of transcriptome exists. Notable amongst the wide array of tools include several for quality visualization (FastQC (\url{http://www.bioinformatics.babraham.ac.uk/projects/fastqc/}) and SolexaQA \citep{Cox:2010ch}) read trimming (e.g. Trimmomatic \citep{Bolger:2014ek} and Cutadapt \citep{Martin:2011va}), read normalization (khmer \citep{Pell:2012id}), assembly (Trinity \citep{Haas:2013jq}, SOAPdenovoTrans \citep{Xie:2013wu}) and assembly verificaton (transrate \url{https://github.com/Blahah/transrate} and RSEM-eval \citep{Li:2014er}).  \\


The ease with which these tools may be used to to produce transcriptome assemblies belies that true complexity underlying the overall process. Indeed, the subtle (and not so subtle) methodological challenges associated with transcriptome reconstruction means that you can easily fuck it up. Amongst the most challenging include isoform reconstruction, simultaneous assembly of low- and high-coverage transcripts, and [] \citep{Modrek:2001ud,Johnson:2003kh}, which together make good transcriptome assembly really difficult. \\


Methodological abuse is widespread. Particularly flagrant are abuses related to quality control of input data, the lack of understanding the role \textit{kmer} selection may play in accurate reconstruction, and lastly, abuses related to the lack of post-assembly quality evaluation. Here, we aim to define a set of evidence based analyses and methods aimed at improving transcriptome assembly, which in turn has significant effects on all downstream analyses. 


To accomplish the proposed standardized methods, we have released a set of version controlled open-sourced code to facilitate this process.   
