\section*{Methods & Results}

To demonstrate the merits of our recommendations, a number of assemblies were produced using a variety of methods. Speciifically, all assembly datasets were produced by asembling a publically available 100bp paired-end Illumina dataset (Short Read Archive ID SRR797058, \citep{Macfarlan:2012js}). This dataset was subsetted randomly into 10, 20, 50, 75, and 100 million read pairs as described in \citep{MacManes:2014io}. Reads were error corrected using the software packare \textsc{bless} version 0.16 \citep{Heo:2014cb} and a kmer=19, which was selected based on the developers recommendation.  Illumina sequencing adapters were removed from both ends of the sequencing reads, as were nucleotides with quality Phred $\leq$ 2, using the program Trimmmatic version 0.32 \citep{Bolger:2014ek}. The adapter and quality trimmed, error corrected reads were then assembled using Trinity release r20140717 or SOADdenovo-Trans version 1.03. Trinity was employed using default settings, while SOAPdenovo-Trans was employed after optimizing kmer size, [and those other flags i forget right now]. \\

For the purposes of demonstrating the efficacy of using multiple assemblers, in this case Trinity and SOAPdenovo-Trans, we merged assemblies via the following process. [Richard fill in details].

For the assembly generated for the illustration of the shortcomings of length based evaluation, we generated an assembly using Trinity that employed settings purposely designed to increase the length of contigs while sacficing accuracy (--path\_reinforcement\_distance 1 --min\_per\_id\_same\_path 80  --max\_diffs\_same\_path 5 --min\_glue 1). \\

All assemblies were characterized using Transrate version 0.31. Using this software, we generated contig metrics, mapping metrics, and comparative metrics which used a the \textit{Mus musculus} version 75 protein file downloaded from Ensembl. All commands for generating the assemblies and downstream analyses are available at [].  